app/assets/javascripts/discourse/app/lib/sticky-avatars.js

```diff
@@ -15,16 +16,16 @@ export default class StickyAvatars {
   direction = "⬇️";
   prevOffset = -1;
 
-  constructor(container) {
-    this.container = container;
+  constructor(owner) {
+    setOwner(this, owner);
```

Comment:
This assumes this class isn't subclassed and/or `this.container` isn't meant to be public